doc/modules/changes.h

```diff
@@ -186,5 +186,11 @@
  * <br>
  * (Juliane Dannberg, 2015/05/20)
  *
+ * <li> New: A new material averaging option using logarithms based on 10 is added.
```

Comment:
replace "logarithms based on 10" by simply "logarithms" -- the base is not important (and the proper technical term would be "logarithm to base 10").